video_xblock/static/js/studio-edit-transcripts-manual-upload.js

```diff
@@ -83,6 +100,59 @@ function validateTranscripts(e, $langChoiceItem) {
 }
 
 /**
+ * Validate extension, name, and size of a transcript or a caption file before save it to video xblock.
+ *
+ * Returns:
+ * isValidated (Boolean): Result of a validation (true|false).
+ *
+ */
+function validateTranscriptFile(event, fieldName, filename, $fileUploader) {
+    'use strict';
+    // User may upload a file without extension. Reference: http://stackoverflow.com/a/1203361
+    var fileExtension = filename.split('.').pop();
+    var fileSize = $fileUploader[0].files[0].size;
+    var acceptedFormats = $fileUploader[0].accept || '.vtt .srt';
+    var isEmptyExtension = fileExtension === '';
+    var isNotAcceptedExtension = acceptedFormats.indexOf(fileExtension) === -1;
+    var isNotAcceptedFormat = isEmptyExtension || isNotAcceptedExtension;
+    // The maximum file size allowed is 300 KB
+    var maxFileSize = 307200;
+    var isNotAcceptedSize = fileSize > maxFileSize;
+    var isNotAccepted = false;
```

Comment:
In any case `isNotAccepted` is equal to `isValidated`
Please delete this rudiment